cygnus-twitter/src/main/java/com/telefonica/iot/cygnus/sinks/TwitterSink.java

```diff
@@ -0,0 +1,482 @@
+/**
```

Comment:
Perfect. Then, the comment below is of highly importance for me. As said, I'll create an issue for discussing it for being solved in a different PR.